src/hooks/useExistingRooms.ts

```diff
@@ -67,11 +76,38 @@ export default function useExistingRooms() {
                     if (hasPermission) {
                         continue
                     }
+
                     await invite({
                         invitees: [sessionAccount!],
                         stream,
                     })
 
+                    // check for an invite-accept and notify the metadata partition
+                    const streamAddress = stream.id.split('/')[0]
+                    if (streamAddress !== account) {
+                        const hasPermission = await stream.hasUserPermission(
+                            StreamPermission.SUBSCRIBE,
+                            sessionAccount!
+                        )
+
+                        if (hasPermission) {
+                            send(MetadataType.AcceptInvite, {
+                                streamPartition: Partition.Metadata,
+                                streamId: stream.id,
+                                data: account,
+                            })
+                            console.log(
+                                'called send with params',
+                                MetadataType.AcceptInvite,
+                                {
+                                    streamPartition: Partition.Metadata,
+                                    streamId: stream.id,
+                                    data: account,
+                                }
+                            )
```

Comment:
You can throw that into the actual `send`, maybe? This way if those logs are helpful to you you'll have them on each `send` hit.

Reply:
This will also dedupe this and similar pieces of code.